build.psm1

```diff
@@ -973,10 +975,17 @@ function Install-Dotnet {
 }
 
 function Get-RedHatPackageManager {
+    param(
+        [Parameter(Mandatory=$true)]
+        [switch]
+        $install
```

Comment:
I think it's better to make each check-in self-contained. I suggest to remove this parameter in this PR, and you can add it back in the next PR when the installation logic is moved here.